src/routes/home/HomeView.vue

```diff
@@ -25,48 +25,80 @@
               </v-btn-toggle>
             </div>
             <v-divider v-if="$vuetify.display.mdAndDown" color="surface-1" class="border-opacity-100" />
-            <v-window v-model="tab" class="pa-4 overflow-y-auto">
-              <v-window-item :value="TABS.PLAY">
-                <p v-if="playableGameList.length === 0" data-cy="text-if-no-game" class="text-surface-1">
-                  {{ t('home.noGameslist') }}
-                </p>
-                <div v-for="game in playableGameList" :key="game.id">
-                  <GameListItem
-                    :name="game.name"
-                    :p0ready="game.p0Ready ? 1 : 0"
-                    :p1ready="game.p1Ready ? 1 : 0"
-                    :game-id="game.id"
-                    :status="game.status"
-                    :num-players="game.numPlayers"
-                    :is-ranked="game.isRanked"
-                    @error="handleSubscribeError(game.id, $event)"
-                  />
-                </div>
-              </v-window-item>
-              <v-window-item :value="TABS.SPECTATE">
-                <p
-                  v-if="spectateGameList.length === 0"
-                  data-cy="no-spectate-game-text"
-                  class="text-surface-1"
-                >
-                  {{ t('home.noSpectatelist') }}
-                </p>
-                <div v-for="game in spectateGameList" :key="game.id">
-                  <GameListItem
-                    :name="game.name"
-                    :p0ready="game.p0Ready ? 1 : 0"
-                    :p1ready="game.p1Ready ? 1 : 0"
-                    :game-id="game.id"
-                    :status="game.status"
-                    :num-players="game.numPlayers"
-                    :is-ranked="game.isRanked"
-                    :is-spectatable="true"
-                    :disable-spectate="game.isOver"
-                    @error="handleError"
-                  />
-                </div>
-              </v-window-item>
-            </v-window>
+            <template v-if="loadingData">
+              <div class="mx-3">
```

Comment:
I think we can probably just put the v-if on the div and remove the template

Reply:
![Screenshot from 2023-11-29 10-40-16](https://github.com/cuttle-cards/cuttle/assets/7325003/4ba66eb5-4385-4f42-a7fa-2605ca49bffb)

Reply:
![Screenshot from 2023-11-29 10-40-46](https://github.com/cuttle-cards/cuttle/assets/7325003/76685bf6-26e8-49c5-942d-2deaa7b8602b)